test/integration/continueOnError/emptyObject/input.json

```diff
@@ -10,7 +10,8 @@
           "type": "string"
         },
         "country": {
-          "type": "object"
+          "type": "object",
+          "properties": {}
```

Comment:
Could we also add an example test in:

https://github.com/thedumbterminal/jsonschema-bigquery/tree/master/test/integration/samples

That shows the input JSON schema and the output bigquery schema working with this new feature.

Reply:
sure. Test added.